test/unit/client-daemonset.bats

```diff
@@ -1206,3 +1206,26 @@ rollingUpdate:
   local actual=$(echo $security_context | jq -r .privileged)
   [ "${actual}" = "true" ]
 }
+
+#--------------------------------------------------------------------
+# client.disableHostNodeID
+
+@test "client/DaemonSet: disableHostNodeID defaults to false" {
+  local actual=$(helm template \
```

Comment:
I think this test and the one below are just missing "cd `chart-dir`" which will allow it to run from a directory other than the root of the repo.
```suggestion
  cd `chart_dir`
  local actual=$(helm template \
```